src/utils/collection.ts

```diff
@@ -191,12 +191,11 @@ export function generateCollectionInsert(collection: ResolvedCollection, data: P
   })
 
   const valuesHash = hash(values)
-  if (opts.hashColumn !== false) {
-    values.push(`'${valuesHash}'`)
-  }
+
+  const valuesWithHash = opts.hashColumn !== false ? [...values, `'${valuesHash}'`] : values
```

Comment:
I'm thinking of removing `hashColumn` option and add it to all, it will simplify the logic here.

I added this option to avoid adding `__hash__` to the development cache table, which causes SQL to fail. But forcing cache cleanup in the new version would be a better solution.